lib/dox/printers/example_request_printer.rb

```diff
@@ -0,0 +1,59 @@
+require 'rexml/document'
```

Comment:
this might not be needed

Reply:
But you are not using this here. This should be in this file https://github.com/infinum/dox/pull/46/files#diff-1ffc32d83c45d7cc42f1091d2ebbce03R1